modules/nextflow/src/main/groovy/nextflow/executor/BridgeExecutor.groovy

```diff
@@ -0,0 +1,179 @@
+            result << '-q' << (task.config.queue.toString())
+        }
+
+        // other cluster options 
+        if( task.config.clusterOptions ) {
+            result << task.config.clusterOptions.toString() << ''
+        }
+
+        return result
+    }
+
+    String getHeaderToken() { '#MSUB' }
+
+    /**
+     * The command line to submit this job
+     *
+     * @param task The {@link TaskRun} instance to submit for execution to the cluster
+     * @param scriptFile The file containing the job launcher script
+     * @return A list representing the submit command line
+     */
+    @Override
+    List<String> getSubmitCommandLine(TaskRun task, Path scriptFile ) {
+
+        ['ccc_msub', scriptFile.getName()]
+
+    }
+
+    /**
+     * Parse the string returned by the {@code sbatch} command and extract the job ID string
+     *
+     * @param text The string returned when submitting the job
+     * @return The actual job ID string
+     */
+    @Override
+    def parseJobId(String text) {
+
+        for( String line : text.readLines() ) {
+            def m = SUBMIT_REGEX.matcher(line)
+            if( m.find() ) {
+                return m.group(1).toString()
+            }
+        }
+
+        def id = text.trim()
+        if( id.isLong() )
+            return id
+
+        throw new IllegalStateException("Invalid BRIDGE submit response:\n$text\n\n")
+    }
+
+    @Override
+    protected List<String> getKillCommand() { ['ccc_mdel'] }
+
+    @Override
+    protected List<String> queueStatusCommand(Object queue) {
+
+        final result = ['squeue','--noheader','-o','%i %t', '-t', 'all']
+
+        if( queue )
+            result << '-p' << queue.toString()
+
+        final user = System.getProperty('user.name')
+        if( user )
+            result << '-u' << user
+        else
+            log.debug "Cannot retrieve current user"
+
+        return result
+    }
+
+    /*
+     *  Maps job status to nextflow status
+     */
+    static private Map STATUS_MAP = [
+            'PD': QueueStatus.PENDING,  // (pending)
+            'R': QueueStatus.RUNNING,   // (running)
+            'CA': QueueStatus.ERROR,    // (cancelled)
+            'CF': QueueStatus.PENDING,  // (configuring)
+            'CG': QueueStatus.RUNNING,  // (completing)
+            'CD': QueueStatus.DONE,     // (completed)
+            'F': QueueStatus.ERROR,     // (failed),
+            'TO': QueueStatus.ERROR,    // (timeout),
+            'NF': QueueStatus.ERROR,    // (node failure)
+            'S': QueueStatus.HOLD,      // (job suspended)
+            'ST': QueueStatus.HOLD,     // (stopped)
+            'PR': QueueStatus.ERROR,    // (Job terminated due to preemption)
+            'BF': QueueStatus.ERROR,    // (boot fail, Job terminated due to launch failure)
+    ]
+
+    @Override
+    protected Map<String, QueueStatus> parseQueueStatus(String text) {
+
+        def result = [:]
+
+        text.eachLine { String line ->
```

Comment:
```suggestion
        for( String line : text.readLines() ) { 
```

Better to user `for` whenever possible, because it avoids the use of intermediate classes required by the closure